katalog/dual-nginx/external/patch/daemonset.yml

```diff
@@ -4,7 +4,7 @@
 
 - op: "replace"
   path: "/spec/template/spec/containers/0/args/1"
-  value: "--configmap=$(POD_NAMESPACE)/nginx-configuration-external"
+  value: "--configmap=$(POD_NAMESPACE)/ingress-nginx-controller-external"
```

Comment:
(see comment for the file)
for example, the index for this argument is no more `0`, is `5` now.